alternator/executor.cc

```diff
@@ -278,6 +281,11 @@ get_table_or_view(service::storage_proxy& proxy, const rjson::value& request) {
         try {
             return { proxy.data_dictionary().find_schema(sstring(internal_ks_name), sstring(internal_table_name)), type };
         } catch (data_dictionary::no_such_column_family&) {
+            // DynamoDB returns validation error even when table does not exist
+            // and the table name is invalid.
+            validate_table_name(table_name);
+
+            // if table_name is valid, raise resource not found exception.
```

Comment:
no need to put comments when the code is clear (I'd say throw meaning is clear for readers familiar with c++)

Reply:
Sure! Removed!